.github/workflows/publish-artifacts.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Admin Plugin Artifact Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to publish from'
+        required: false
+        default: ''
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      # Step 1: Checkout the repository
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.branch || github.ref }} # ✅ Use manual input if provided, fallback to current branch
```

Comment:
Why manual input? Shouldn't we ensure that the current ref has a tag that looks like a version?